packages/bits/src/lib/table/table.component.ts

```diff
@@ -93,8 +110,23 @@ export class TableComponent<T> extends CdkTable<T> implements OnInit, AfterViewI
                 protected _dir: Directionality,
                 private tableStateHandlerService: TableStateHandlerService,
                 @Inject(DOCUMENT) private document: Document,
-                private platform: Platform) {
-        super(_differs, _changeDetectorRef, _elementRef, role, _dir, document, platform);
+                private platform: Platform,
+                @Inject(_VIEW_REPEATER_STRATEGY) viewRepeater: _ViewRepeater<T, RenderRow<T>, RowContext<T>>,
+                @Inject(_COALESCED_STYLE_SCHEDULER) coalescedStyleScheduler: _CoalescedStyleScheduler,
+                viewportRuler: ViewportRuler,
+                @Optional() @SkipSelf() @Inject(STICKY_POSITIONING_LISTENER) stickyPositioningListener: StickyPositioningListener
+    ) {
+        // the parameters that were added to the constructor were optional before v12
+        super(
+            _differs, _changeDetectorRef, _elementRef, role, _dir, document, platform,
+            viewRepeater, coalescedStyleScheduler, undefined, undefined
+        );
+
+        // v12 is changing the order of the last two parameters compared to v11. They deprecated the optionality of the last parameter, but the one before
```

Comment:
Maybe just for clarity:
```suggestion
        // Angular v12 is changing the order of the last two parameters compared to v11. They deprecated the optionality of the last parameter, but the one before
```